cellprofiler/modules/measureimageareaoccupied.py

```diff
@@ -215,13 +216,20 @@ def display(self, workspace, figure):
     def measure_objects(self, operand, workspace):
         '''Performs the measurements on the requested objects'''
         objects = workspace.get_objects(operand.operand_objects.value)
+        dimensions = len(objects.shape)
         if objects.has_parent_image:
```

Comment:
I don’t understand this branch.

Why should we care whether a `cellprofiler.object.Objects` instance has a `parent_image`?